jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -337,6 +380,55 @@ YUI.add('deployment-flow', function() {
     },
 
     /**
+      Get the list of terms that the user has already agreed to.
+
+      @method _getAgreements
+    */
+    _getAgreements: function() {
```

Comment:
I feel like this isn't being tested....did you miss a test or am I missing where it's tested?